prql-compiler/src/sql/srq/anchor.rs

```diff
@@ -226,7 +224,7 @@ pub(super) fn anchor_split(
     ctx: &mut AnchorContext,
     preceding: Vec<SqlTransform<TableRef>>,
     atomic: Vec<SqlTransform<TableRef>>,
-) -> (Vec<SqlTransform<TableRef>>, HashMap<CId, CId>) {
+) -> Vec<SqlTransform<TableRef>> {
```

Comment:
(fwiw I do think simplifying signatures is good per se)

Reply:
Yeah, because I store cid_redirects in Context. Which is more complex I'd say.